Configuration/TCA/tx_styleguide_elements_basic.php

```diff
@@ -871,6 +871,42 @@
             ],
         ],
 
+        'text_21' => [
+            'label' => 'text_21',
+            'description' => 'renderType=textTable',
+            'config' => [
+                'type' => 'text',
+                'renderType' => 'textTable',
+                'cols' => 40,
+                'rows' => 5,
+                'fieldControl' => [
+                    'tableWizard' => [
+                        'options' => [
+                            'numNewRows' => 3,
+                        ],
+                    ],
+                ],
+            ],
+        ],
+        // It doesn't seem to work like that yet, only an empty wizard is shown
+        'text_22' => [
```

Comment:
I'd prefer to leave out this example if it's broken.
Please rebase patch to origin/master, too ... or fix access rights on your repo.